src/containerapp/azext_containerapp/custom.py

```diff
@@ -3656,6 +3660,72 @@ def disable_dapr(cmd, name, resource_group_name, no_wait=False):
     except Exception as e:
         handle_raw_exception(e)
 
+def init_dapr_component(cmd, resource_group_name, environment_name):
+    _validate_subscription_registered(cmd, CONTAINER_APPS_RP)
+
+    # Create the Redis service
+    redis_capp_name = DAPR_REDIS_PREFIX +'-' + ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(5))
```

Comment:
What happens when you run `dapr init` multiple times? I think the ACA behavior should be similar. My expectation would be that it's a no-op if it's been run already on the ACA environment (or it could confirm components are correctly installed and repair). Perhaps we can create the Redis service using a fixed name that is unlikely to collide with something the customer would create. We could then check for its existence. Having a fixed naming scheme could make it easier to implement an "un-init" command too.

Reply:
Right, let us call it `dapr-redis` only, and all the operations are `PATCH`, so it becomes a no-op.